<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComyFletes</class>
 <widget class="QDialog" name="ComyFletes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_agregar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+ Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_regresar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-  Regresar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Monto Bruto</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_cel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cel. Obra</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_transeq">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Trans. eq. menor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_auto">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Obra</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_cam">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Camioneta Obra</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_celobra">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_transeq">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_auto">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_camioneta">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
